config/serverless-operator.yaml

```diff
@@ -0,0 +1,73 @@
+# Full struct in cmd/prowgen/prowgen.go#Config
+
+config:
+  branches:
+    main:
+      openShiftVersions:
+        - version: 4.13
+          cron: "0 4,16 * * *"
```

Comment:
Maybe we could reduce the runs at this point to just "once a day" ? Running twice a day is probably not necessary.